scripts/install

```diff
@@ -126,6 +128,54 @@ gpasswd --add www-data $app
 #=================================================
 ynh_script_progression --message="Setting up source files..."
 
+if [ $(uname -m) == "armv7l" ] 
+then
+	# prepare openjdk8 installation
+	if  [ -d "/tmp/jitsi-install/" ]
+	then
+		rm -rf /tmp/jitsi-install/
+	fi
+	
+	mkdir /tmp/jitsi-install/
+	#install openjdk 8 because javah is used by jitsi and not supported in higher versions
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jre_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jre-headless_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jdk_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jdk-headless_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	
+	sudo apt install \
+	 /tmp/jitsi-install/openjdk-8-jre_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jre-headless_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jdk_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jdk-headless_8u322-b06-1~deb9u1_armhf.deb
+	
+	# ser openjdk as default 
+	sudo update-alternatives --set java /usr/lib/jvm/java-8-openjdk-armhf/jre/bin/java
+	
+	# stop related services if exist
+	#sudo systemctl stop prosody jitsi-videobridge jitsi-jicofo
+
+	# prepare jniwrapper comppilation
+	export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-armhf
+	sudo apt update
+	sudo apt install automake autoconf build-essential libtool git maven m4 -y
+	git clone https://github.com/sctplab/usrsctp /tmp/jitsi-install/usrsctp
+	git clone https://github.com/jitsi/jitsi-sctp /tmp/jitsi-install/jitsi-sctp
```

Comment:
not fan of git clone, if there are new versions of the main branch, it can break the installation

Reply:
sorry im not really friendly with that, i just integrate steps founded on a documentation.
do you have any other  way to do that ?

Reply:
usually we use an app.src file like it's done for https://github.com/YunoHost-Apps/jitsi_ynh/blob/master/conf/jitsi-meet-prosody.src